relay/sources/relayd/Cargo.toml

```diff
@@ -23,41 +23,42 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 nom = "5.0"
 # Could use native lib instead, but maybe not prod-ready
-flate2 = { version = "1.0", features = ["zlib"], default-features = false }
-diesel = { version = "1.4", features = ["postgres", "chrono", "r2d2"] }
+flate2 = { version = "1.0.11", features = ["zlib"], default-features = false }
```

Comment:
Same for all other dependencies, why adding a precise version (which is already fixed in Cargo.lock)